src/Hooks/useLoginForm.tsx

```diff
@@ -0,0 +1,54 @@
+import {useState, useEffect} from 'react';
+
+const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+export const useLoginForm = () => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [emailError, setemailError] = useState('');
```

Comment:
```suggestion
    const [emailError, setEmailError] = useState('');
```

camelCase 규칙을 잘 지켜주세요 😁